csrc/python_frontend/python_bindings.cpp

```diff
@@ -3148,6 +3148,41 @@ void initNvFuserPythonBindings(PyObject* module) {
       },
       py::arg("arg"),
       py::arg("dim"));
+  nvf_sched.def(
+      "reduction_factor",
+      [](FusionDefinition::SchedOperators& self,
+         Tensor arg,
+         std::vector<int>& dims) -> Tensor {
+        FUSER_PERF_SCOPE("SchedOperators.reduction_factor");
+        TORCH_CHECK(
+            self.validUse(),
+            "Attempting to use a SchedOperators Op prior to definition!");
+        FusionDefinition* fd = self.fusion_definition;
+        auto input_tv =
+            fd->getFusionState(arg.index)->template as<TensorView>();
+        auto output_tv = input_tv->rFactor(dims);
+        Tensor output = fd->defineTensor(arg.dims);
+        fd->addFusionState(output.index, output_tv);
+        return output;
+      },
+      py::arg("arg"),
+      py::arg("dims"));
+  nvf_sched.def(
+      "reorder",
+      [](FusionDefinition::SchedOperators& self,
+         Tensor arg,
+         std::unordered_map<int, int>& old2new) {
+        FUSER_PERF_SCOPE("SchedOperators.reorder");
+        TORCH_CHECK(
+            self.validUse(),
+            "Attempting to use a SchedOperators Op prior to definition!");
+        FusionDefinition* fd = self.fusion_definition;
```

Comment:
We can check length of old2new here and perhaps check for repeats and that max value is less than dim.

Reply:
The `old2new` is already checked inside of the C++ code. I did add testing to confirm that is the case.